rmw_connext_cpp/src/rmw_publisher.cpp

```diff
@@ -28,9 +28,10 @@
 
 #include "rmw_connext_cpp/identifier.hpp"
 
+#include "connext_static_publisher_info.hpp"
 #include "process_topic_and_service_names.hpp"
 #include "type_support_common.hpp"
-#include "rmw_connext_cpp/connext_static_publisher_info.hpp"
+
```

Comment:
Nitpick: delete this empty line?

Reply:
Yes! An oversight. See https://github.com/ros2/rmw_connext/pull/447/commits/ed082c3c8b4c867d8845e82be184dfbdef6f54dd